lib/workers/repository/finalise/repository-statistics.ts

```diff
@@ -33,3 +35,55 @@ export function runRenovateRepoStats(
   }
   logger.debug({ stats: prStats }, `Renovate repository PR statistics`);
 }
+
+interface BranchMetadata {
+  branchName: string;
+  sha: string | null;
+  parentSha: string | null;
+  automerge: boolean;
+  isModified: boolean;
+}
+
+interface BaseBranchMetadata {
+  branchName: string;
+  sha: string;
+}
+
+export function runBranchSummery(): void {
+  const { scan, branches } = getCache();
+
+  const baseMetadata: BaseBranchMetadata[] = [];
+  for (const [branchName, cached] of Object.entries(scan ?? {})) {
```

Comment:
u can use map i believe here

Reply:
Problem is that it becomes unreadable, as `scan` is of type `Record` therefore `Object.entires(scan)` returns an array of tuples, where tuple[0] is the key, tuple[1] is the value and I need both.

so I think the `for .. of` approach is much more readable

Reply:
well looks good anyway :D